frontend/website-redesign/src/components/Contributors.re

```diff
@@ -1,123 +1,100 @@
 module Styles = {
   open Css;
-  let container =
-    style([
-      padding2(~v=`rem(4.), ~h=`rem(1.25)),
-      media(
-        Theme.MediaQuery.desktop,
-        [
-          padding2(~v=`rem(8.), ~h=`rem(9.5)),
-          maxWidth(`rem(89.)),
-          margin2(~v=`zero, ~h=`auto),
-        ],
-      ),
-    ]);
-  let header = merge([Theme.Type.h2, style([marginBottom(`rem(0.5))])]);
-  let sectionSubhead =
-    merge([
-      Theme.Type.sectionSubhead,
-      style([
-        fontSize(`px(19)),
-        lineHeight(`rem(1.75)),
-        marginBottom(`rem(2.93)),
-        letterSpacing(`pxFloat(-0.4)),
-      ]),
-    ]);
-  let headerCopy =
-    style([media(Theme.MediaQuery.desktop, [width(`rem(42.))])]);
+  let container = style([margin2(~v=`rem(4.), ~h=`zero)]);
+
   let genesisRule =
     style([media(Theme.MediaQuery.desktop, [width(`percent(100.))])]);
-};
 
-module GenesisMembersGrid = {
-  module Styles = {
-    open Css;
+  let modalContainer = modalShowing =>
+    style([
+      modalShowing ? opacity(1.) : opacity(0.),
+      modalShowing ? pointerEvents(`auto) : pointerEvents(`none),
+      transition("opacity", ~duration=400, ~timingFunction=`easeIn),
+      position(`fixed),
+      height(`vh(100.)),
+      width(`vw(100.)),
+      backgroundColor(`rgba((0, 0, 0, 0.3))),
+      display(`flex),
+      top(`percent(50.)),
+      left(`percent(50.)),
+      transform(`translate((`percent(-50.), `percent(-50.)))),
+      zIndex(2),
+    ]);
 
-    let genesisHeader = merge([Theme.Type.h2, style([])]);
-    let genesisCopy =
-      style([
-        unsafe("grid-area", "1 /1 / span 1 / span 2"),
-        media(Theme.MediaQuery.tablet, [width(`rem(34.))]),
-      ]);
-    let sectionSubhead =
-      merge([
-        Theme.Type.sectionSubhead,
-        style([
-          fontSize(`px(19)),
-          lineHeight(`rem(1.75)),
-          marginTop(`rem(0.5)),
-          marginBottom(`rem(2.)),
-          letterSpacing(`pxFloat(-0.4)),
-        ]),
-      ]);
-    let grid =
-      style([
-        marginTop(`rem(1.)),
-        display(`grid),
-        paddingTop(`rem(1.)),
-        gridTemplateColumns([`rem(10.), `rem(10.)]),
-        gridAutoRows(`rem(17.3)),
-        gridColumnGap(`rem(1.)),
-        gridRowGap(`rem(1.)),
-        media(
-          Theme.MediaQuery.tablet,
-          [gridTemplateColumns([`repeat((`num(3), `rem(10.)))])],
-        ),
-        media(
-          Theme.MediaQuery.desktop,
-          [gridTemplateColumns([`repeat((`num(5), `rem(11.)))])],
-        ),
-      ]);
-  };
+  let modal = style([margin(`auto)]);
+};
+
+module MemberModal = {
```

Comment:
This probably doesn't need to be in a separate module, you can just take put the return value of `make` directly into the main component.